docs/extensions/storage/testing.md

```diff
@@ -84,6 +84,7 @@ TEST_SERVER_URL=https://localhost:9200 \
 TEST_EXTERNAL_USER_BACKENDS=true \
 TEST_OCIS=true \
 OCIS_REVA_DATA_ROOT=/var/tmp/ocis/owncloud/ \
+DELETE_USER_DATA_CMD='rm -rf /var/tmp/ocis/storage/users/nodes/root/* /var/tmp/ocis/storage/users/nodes/*-*-*-*' \
```

Comment:
This file mentions "the ocis-reva code" and "Run a ldap server in a docker container"... It came automagically from the `ocis-reva` repo during the mono-repo changes.

Why would we keep it? Delete the whole file?

Reply:
you are right, I don't think we need to run tests on that extension by itself, I've deleted the file in 2f4b3e47cd836a9528b9f11724e44a1b3766f5b0